convert-hf-to-gguf.py

```diff
@@ -80,19 +84,42 @@ def __init__(self, dir_model: Path, ftype: gguf.LlamaFileType, fname_out: Path,
         self.block_count = self.find_hparam(["n_layers", "num_hidden_layers", "n_layer"])
         self.tensor_map = gguf.get_tensor_name_map(self.model_arch, self.block_count)
         self.tensor_names = None
+        self.metadata = metadata
+
+        model_tensors = self.get_tensors()
+
         if self.ftype == gguf.LlamaFileType.GUESSED:
             # NOTE: can't use field "torch_dtype" in config.json, because some finetunes lie.
-            _, first_tensor = next(self.get_tensors())
+            _, first_tensor = next(model_tensors)
```

Comment:
`next` consumes an element from the generator expression returned by `self.get_tensors()`. If it's re-used after, the first tensor will be missing from what it returns.

Do not store `self.get_tensors()` into a variable like this if you want to start from the beginning when counting the parameters.